src/test/java/org/folio/rest/camunda/utility/Parameters.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.rest.camunda.utility;
+
```

Comment:
Could you add Javadocs describing the purpose of this test utility class?

Reply:
Done.